converters/toZigbee.js

```diff
@@ -3331,6 +3331,12 @@ const converters = {
             }
         },
     },
+    RM01_row_button: {
+        key: ['rm01_row_2', 'rm01_row_3', 'rm01_row_4'],
+        convertSet: async (entity, key, value, meta) => {
+            return;
```

Comment:
This converter is empty, why is it needed?

Reply:
i dont know a another solution.
My Idea, The fromZigbee function are:
```

RM01_on_click: {
        cluster: 'genOnOff',
        type: 'commandOn',
        options: [exposes.options.legacy()],
        convert: (model, msg, publish, options, meta) => {
            if (isLegacyEnabled(options)) {
                const button = getKey(model.endpoint(msg.device), msg.endpoint.ID);
                return {action: `${button}_on`};
            } else {
                return fromZigbeeConverters.command_on.convert(model, msg, publish, options, meta);
            }
        },
    },


```
In legacy, the command send an 'rm01_row_2_on' or 'rm01_row_2_off'. I i send to zigbee (from ioBroker or somethink like that) 'rm01_row_2_on' or 'rm01_row_2_off' it come to an error like:

` '2021-10-14 14:53:11.048 | error | No converter available for '6735/6736/6737' with key 'rm01_row_2'
`
But the device work so, that always push the button it send always the on oder off command. The device don´t store the status from to the second row